scipy/linalg/basic.py

```diff
@@ -1389,12 +1386,9 @@ def pinvh(a, cond=None, rcond=None, lower=True, return_rank=False,
     a : (N, N) array_like
         Real symmetric or complex hermetian matrix to be pseudo-inverted
     cond, rcond : float or None
-        Cutoff for 'small' singular values.
-        Singular values smaller than ``rcond*largest_singular_value``
-        are considered zero.
-        If None and the dtype of `a` is ``np.float32``, it is set to
-        ``np.finfo('float32').eps * 1e3``.
-        Otherwise, it is set to ``np.finfo('float64').eps * 1e6``.
+        Cutoff for 'small' singular values. If omitted, singular values smaller
+        than ``max(M,N)*largest_singular_value*eps`` are considered zero where
+        ``eps`` is the machine precision.
```

Comment:
This could use a versionchanged directive

Reply:
Will add.